src/components/RecipeEditView.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { Grid, Paper, Button } from '@mui/material'
```

Comment:
We use styled components and a component library called Suitcase, but I think it is fine to use a library for this as long as you know how styled_components are used.

Reply:
Awesome!
Yeah, I read that in the onboarding docs after I was done with this PR :facepalm: